Tools/WinMLRunner/src/main.cpp

```diff
@@ -0,0 +1,8 @@
+#include "CommandLineArgs.h"
+#include "Run.h"
+
+int main(int argc, char** argv)
```

Comment:
Is this a main function for winmlrunner.exe? are we using argv here?

Reply:
You're right, this is the main function. Fixed!